internal/common/resource.go

```diff
@@ -490,6 +491,9 @@ func defaultUpdateFunc(newObj, foundObj client.Object) {
 	case *promv1.ServiceMonitor:
 		foundObj.(*promv1.ServiceMonitor).Spec = newTyped.Spec
 
+	case *tekton.Task:
```

Comment:
Pipelines should be here too?

Reply:
I didn't include it because if it has some uncommon logic, so it would be better to keep it as it is.

Reply:
ok